cpp/data/eigerfan/src/MultiPullBroker.cpp

```diff
@@ -47,25 +47,23 @@ void MultiPullBroker::connect(std::string& endpoint, void* inproc_context) {
  * \param[in] endpoint Endpoint of socket to pull data from
  */
 void MultiPullBroker::worker_loop(std::string& endpoint) {
-  int hwm = 10000;
-  int linger = 100;
 
   // Create source in new isolated context
   // It is important to create a new context in each worker thread, as there are
   // throughput limitations to a context shared between threads. Increasing ZMQ IO
   // threads on the context is not sufficient.
   zmq::context_t source_context(1);
   zmq::socket_t source_socket(source_context, ZMQ_PULL);
-  source_socket.setsockopt(ZMQ_SNDHWM, &hwm, sizeof(hwm));
-  source_socket.setsockopt(ZMQ_LINGER, &linger, sizeof(linger));
+  source_socket.setsockopt(ZMQ_SNDHWM, &SEND_HWM, sizeof(SEND_HWM));
```

Comment:
We should have a different, lower HWM for the worker threads. This changes it from 10000 to 100000, which is the same as the main rx thread.

Reply:
I've created a WORKER_SEND_HWM to differentiate

Reply:
Addressed in 0932403